config/lobby/db/001_create_tables

```diff
@@ -1,94 +1,58 @@
-create table bad_words (
-    word character varying(40) not null
+CREATE TABLE ta_users (
+    username character varying(40) NOT NULL PRIMARY KEY,
```

Comment:
I moved the PRIMARY KEY constraints to the columns themeselves

Reply:
No objection, though please stick to all lower case if you would.

Reply:
I personally like having keywords in CAPS and everything else lowercase, because it makes it easy to differenciate between keywords and names, no problem, I can change that.

Reply:
Also it's the default SQL style, so I just stick to it.

Reply:
> Also it's the default SQL style, so I just stick to it.

The help docs all use it, but in those cases the keywords are the important things. It turns out that the keywords are some of the least important things in a sql statement. If you format it well, then it is the table, column and where condition clauses that are the most important. `WHERE` is magnitudes less important than anything that comes after it.